### ISSUE TITLE
Image coordinate calculation currently not correct

### ISSUE THREAD

**Issue by DePrinsT** [benjaminpope/drpangloss#56]
The image coordinate calculation function currently in the `models.py` goes as follows:

```python
def _image_coordinates(npix, fov_mas):
    """Return Cartesian image-plane coordinates in milliarcseconds."""
    coords = np.linspace(-0.5 * fov_mas, 0.5 * fov_mas, int(npix))
    return np.meshgrid(coords, coords, indexing="xy")
```

However, this is not consistent with the usual interferometric conventions of image coordinates, where the image coordinate denote the centres of the pixels and the (x, y) = (0, 0) position always lands on the geometric center of the image  (i.e. given an uneven amount of pixels (0, 0) corresponds to the centremost pixels, but for an even amount of pixels it corresponds to the midpoint between the 4 centremost pixels). Given an image of an integer amount of pixels and a given floating point pixelscale, the coordinates should hence never land on the edges of the FOV, while this is the case in the current implementation. 

An alternative implementation which does follow the usual convention (both for even and uneven amounts of pixels in the image) would be:

```python
def _image_coordinates(npix, fov_mas):
    """Return Cartesian image-plane coordinates in milliarcseconds."""
    # Pixelscale    
    ps_mas = fov_mas / int(npix)

    # Indices of pixels in order of 1D numerical array
    i = jnp.arange(npix)
    j = jnp.arange(npix)

   # 1D coordinates
    x = -(j - (npix - 1) / 2) * ps_mas
    y = ((npix - 1) / 2 - i) * ps_mas

    # 2D meshgrid coordinates
    xx, yy = np.meshgrid(x, y, indexing="xy")
    
    return xx, yy
```

Note that this assumes the results from meshgrid follow standard python/numpy/matplotlib index ordering in 2D image arrays, i.e. the 1st index follows the y-axis, ranging from top to bottom (north to south) and the 2nd index follows the x-axis, ranging from left to right (east to west). Note that this convention makes it so the y-coordinates now range from positive to negative as you range through the array indices, which is incompatible with the previous definition.
